src/pages/Home/Home.js

```diff
@@ -129,7 +129,7 @@ const Home = () => {
 							) : (
 								<>
 									<Slider ref={sliderRef} {...settings}>
-										{vcEntityList.map((vcEntity, index) => (
+										{vcEntityList.sort((a, b) => b.id - a.id).map((vcEntity, index) => (
```

Comment:
You are right! But, I just realised that the whole sorting procedure is redundant here because it has already been done by the CredentialsContext.js file. So instead, I think I should just use vcEntryList here

Reply:
This means that I will revert this change.